captures the most common parameters and identities in "schedule-status" grouping (#84)
boucadair/policy-based-network-acl: draft-ma-opsawg-schedule-yang.md

   The "schedule-id" parameter is useful to uniquely identify a schedule in
   a network device or controller if multiple scheduling contexts exists.

>    The "state" parameter is defined to configure/expose the sheduling state,

Reviewer: ```suggestion
   The "state" parameter is defined to configure/expose the scheduling state,
```

   depending on the use of the grouping. The "identityref" type is used for this

>    parameter to allow extensibility in future modules. For example, a "conflicted"

Reviewer: ```suggestion
   parameter to allow extensibility in future modules. For example, a "conflict"
```

   state is valid in scheduling contexts where multiple systems struggle for the

>    scheduling of the same property.

Reviewer: ```suggestion
   scheduling of the same property. The conflict may be induced by, e.g., multiple entities managing the schedules for the same target component.
```

>    The "version" parameter summarizes the current schedule version information,

Reviewer: ```suggestion
   The "version" parameter is used to track the current schedule version information. The version can be bumped by the entity who create the schedule.
```

>    and the "last-update" parameter identifies when the schedule was last modified.

Reviewer: ```suggestion
   The "last-update" parameter identifies when the schedule was last modified. In some contexts, this parameter can be
   used to track the configuration of a given schedule. In such cases, the "version" is not used.
```

Author: > In such cases, the "version" is not used.

In such cases, the "version" may not be used. 
Better? i think these two parameters are  not conflicting. An iteration version number helps obtain the number of historical modifications, while last-update can be used to understand when it was last modified.

Reviewer: Ok for "may not be" for now. Edited the change  accordingly. 

I expect we will have this comment during the publication process to clarify the relationship between the two.

Author: Okay, sure, I am okay to drop "version" parameter if there is any objection in the future.

   The "schedule-type" parameter identifies the type of the current schedule.
   The "counter", "last-occurrence", and "upcoming-occurrence" data nodes are
   only avaliable when the "schedule-type" is "recurrence".

   The current grouping captures the most common definition that is applicable

>    to all scheduling contexts. Future modules can define other useful

Reviewer: ```suggestion
   to typical scheduling contexts known so far. Future modules can define other useful
```

>    parameters as needed. For example, in scheduling context that allows multiple

Reviewer: ```suggestion
   parameters as needed. For example, in a  scheduling context with multiple
```

>    system sources to exist, the "source" and "precedence" parameters may need to

Reviewer: ```suggestion
   system sources to feed the schedules, the "source" and "precedence" parameters may need to
```
